independent-projects/resteasy-reactive/server/runtime/pom.xml

```diff
@@ -19,6 +19,10 @@
             <groupId>io.quarkus.resteasy.reactive</groupId>
             <artifactId>resteasy-reactive-common</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Do we really want this? I'm not against it, obviously, but it's a big change. 

@geoand

Reply:
We do **not** want it. 

I to Franz that the core module does not depend on Vert.x and I don't want to add such a dependency just for the sake of this change